test_graphyte.py

```diff
@@ -141,6 +142,38 @@ def test_timestamp_generated(self):
         timestamp = int(match.group(1))
         self.assertTrue(send_time - 2 <= timestamp <= send_time + 2)
 
+    def test_send_socket_do_raise_error(self):
+
+        class SenderWithError(graphyte.Sender):
+            def __init__(self, *args, **kwargs):
+                graphyte.Sender.__init__(self, 'dummy_host', *args, **kwargs)
+
+            def send_message(self, message):
+                raise ConnectionError()
```

Comment:
I don't think `ConnectionError` exists on Python 2.7. How about just `RuntimeError` or something generic, seeing we're catching everything?